nestjs/src/courses/mentor-reviews/mentor-reviews.controller.ts

```diff
@@ -30,4 +34,12 @@ export class MentorReviewsController {
 
     return new MentorReviewsDto(mentorReviews);
   }
+
+  @Post('/')
+  @ApiOperation({ operationId: 'assignReviewer' })
+  @ApiOkResponse({})
+  @RequiredRoles([CourseRole.Dementor, CourseRole.Manager, Role.Admin], true)
```

Comment:
Let's make this functionality available only to managers and admins. The role of Dementors should only have the ability to view who has checked. And on the frontend, the button should also be accessible only to admins and managers.